pages/_app.tsx

```diff
@@ -36,41 +36,18 @@ export type OmitFirstTwoChars<S extends string> = S extends `${infer _}${infer _
 export type ColorNames = OmitFirstTwoChars<keyof typeof CSSVariableColors>;
 
 export default function App({ Component, pageProps }) {
-  const currentPageTitle = typeof window !== "undefined" ?
-    window.location.pathname.substring(1)  // Removing the leading '/'
-      .split('-')  // Splitting by dashes
-      .map(word => word.charAt(0).toUpperCase() + word.slice(1))  // Capitalizing the first letter of each word
-      .join(' ')
-    : "";
-  const titleSuffix = "26-28 June, JSConfBP 2024"
-  const title = currentPageTitle ? `${currentPageTitle} | ${titleSuffix}` : titleSuffix;
-
-  const url = `https://jsconfbp.com${typeof window !== "undefined" ? window.location.pathname : "/"}`;
-  const description = 'JSConf Budapest 2024, June 26-28, Budapest, Hungary. Tickets from €450, including free workshops, inclusive catering and barista coffee!';
-  const socialImgAbsoluteUrl = 'https://jsconfbp.com/social-share.png';
-
   return (
     <>
       <Head>
-        <title>{title}</title>
-        <meta name="description" content={description} key="description" />
         <meta name="keyworkds" content="jsconf, 2024, javascript, jsconfbp, conference, budapest, jsconf budapest, diversity, inclusivity, community" key="keywords" />
         <meta name="theme-color" content="#07bdbd" key="theme-color" />
         <meta name="viewport" content="width=device-width, initial-scale=1" key="viewport" />
         <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon"></link>
-        <link rel="canonical" href={url} />
-        <meta property="og:title" content={title} key="og:title" />
-        <meta property="og:description" content={description} key="og:description" />
-        <meta property="og:url" content={url} key="og:url" />
-        <meta property="og:image" content={socialImgAbsoluteUrl} key="og:image" />
-        <meta property="og:image:alt" content="JSConf Budapest June 26-28, 2024 logo" key="og:image:alt" />
         <meta name="twitter:card" content="summary_large_image" key="twitter:card" />
         <meta name="twitter:site" content="@jsconfbp" key="twitter:site" />
         <meta name="twitter:creator" content="@_gregnagy" key="twitter:creator" />
-        <meta name="twitter:title" content={title} key="twitter:title" />
```

Comment:
Why did you leave social share-related tags here?
Could we move them into SocialShare as well so the component has everything that belongs to this concern?